src/components/SearchResultItem.tsx

```diff
@@ -46,14 +47,14 @@ export class SearchResultItem extends React.Component<SearchResultItemProps, Sea
         if (this.state.selected) {
             let container = ReactDOM.findDOMNode(this.props.libraryContainer);
             let currentItem = ReactDOM.findDOMNode(this);
-            let containerRect = container.getBoundingClientRect();
-            let currentRect = currentItem.getBoundingClientRect();
+            let containerRect = container?.getBoundingClientRect();
+            let currentRect = currentItem?.getBoundingClientRect();
 
-            if (currentRect.top < currentRect.height) {
+            if (currentRect?.top < currentRect?.height) {
```

Comment:
so if these variables are null, what should this operator (<) return ?

Reply:
that's a good question, I can probably just check for null further up and bail.